listener/listener.go

```diff
@@ -151,6 +153,16 @@ func WithSubscriptionDetails(lock time.Duration, maxDelivery int32) ManagementOp
 	}
 }
 
+func WithMessageLockAutoRenewal(interval time.Duration) Option {
```

Comment:
If the interval is greater than lock duration than it will simply do nothing, correct? Seems like ok behavior, but thought I'd call it out in case we want to call it out in a comment.

Reply:
that's correct. if the lock expire after 10 sec, and you auto-renew every 20 second, you're not doing anything.
leaving this as a potential improvement, and good enough for now.